app/code/Magento/Inventory/Model/ResourceModel/Reservation/Collection.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Inventory\Model\ResourceModel\Reservation;
+
+use Magento\Framework\Model\ResourceModel\Db\Collection\AbstractCollection;
+use Magento\Inventory\Model\ResourceModel\Reservation as ReservationResourceModel;
+use Magento\Inventory\Model\Reservation as ReservationModel;
+
+/**
+ * Resource Collection of Reservation entities
+ *
+ * @api
```

Comment:
don't think we need to mark this entity as API 
as the main reason we have it - is for service purposes only